Resources/Prototypes/_Scp/Access/AccessGroup/access_group.yml

```
@@ -0,0 +1,67 @@
+  id: ScpBlackAlertAccesses
+  tags:
+  - HeadOfSecurity
+  - Security
+  - Command
+
+
+
+- type: accessGroup
+  id: ScpDeltaAlertAccesses
+  tags:
+  - HeadOfSecurity
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)